### PR TITLE
Allow having clickable areas one inside another

### PR DESCRIPTION
**Pull request by otommod**
Previously, if you wanted to, say, handle both scroll-up and scroll-down, you couldn't. This patch fixes that.
Some basic testing I did doesn't produce any errors.

### CODE REVIEW

**Comment by otommod** [comment]
I just noticed that this has been an open request (#66) for quite some time, and that there's also a patch there...

File: bar.c [hunk 1]
```diff
@@ -1147,7 +1158,7 @@ main (int argc, char **argv)
                         case XCB_BUTTON_PRESS:
                             press_ev = (xcb_button_press_event_t *)ev;
                             {
-                                area_t *area = area_get(press_ev->event, press_ev->event_x);
+                                area_t *area = area_get(press_ev->event, press_ev->detail, press_ev->event_x);
                                 /* Respond to the click */
                                 if (area && area->button == press_ev->detail) {
```
Comment (mrshankly) [inline bar.c]: `area->button` is already checked in the new `area_get` function, see [here](https://github.com/otommod/bar/blob/cece20b562adf3d30c2f2a071736e82371559472/bar.c#L230). Should this check be kept here?

Comment (LemonBoy) [inline bar.c]: Yep, the check should be omitted

Comment (otommod) [inline bar.c]: Indeed, fixed.

**Comment by LemonBoy** [comment]
@mrshankly So, is this working as intended ?

**Comment by mrshankly** [comment]
> @mrshankly So, is this working as intended ?

Yes, I think so. I've been using it for more than a week now and haven't noticed any problems.

**Comment by LemonBoy** [comment]
Awesome, then I'll just wait for @otommod to rebase the PR to merge it :)
If you feel that something about this should be added to the documentation feel free to write a line or two!

**Comment by otommod** [comment]
@LemonBoy rebased. I'm not too familiar with git yet, so I rebased and force pushed, I don't know if that's considered bad. Maybe I should have merged master instead?

I see you change the comments to C++-style. TBH, I prefer C-style comments, but it's your project, so I changed mine too.

**Comment by andornaut** [comment]
It'd be great if the readme could be updated with the new syntax for multiple actions.

**Comment by LemonBoy** [comment]
@andornaut feel free to do so, I assumed that since many people requested that feature it should be easy for them to make use of the multiple actions.

**Comment by andornaut** [comment]
Done in #104

**Pull request merged**